### DISCUSSION TITLE
Module language

### DISCUSSION THREAD

**Discussion by stepanzak** [abenz1267/walker#34] [Q&A] [answered]
Hi, do modules (plugins) have to be written in JavaScript?

**Comment by abenz1267** [comment, marked as answer]
> Hi, do modules (plugins) have to be written in JavaScript?

No. Walker doesn't care what.

**Reply by stepanzak** [reply]
So the plugin script should print the json into standard output?

**Reply by abenz1267** [reply]
Yes
